cfme/tests/infrastructure/test_config_management.py

```diff
@@ -49,31 +49,50 @@ def tag(category):
 
 
 @pytest.mark.meta(blockers=["BZ#1244842"])
+@pytest.mark.uncollectif(lambda config_manager_obj: config_manager_obj.type == "Ansible Tower" and
```

Comment:
Since we apply this very same uncollectif mark to all the tests here, we can use this simpler https://github.com/RedHatQE/cfme_tests/blob/bfbcd5eb2e2889cc177135f17a36fe355794a87c/utils/tests/test_metadoc.py#L3 approach instead.